happytransformer/happy_next_sentence.py

```diff
@@ -0,0 +1,78 @@
+import torch
+import re
+from transformers import (
+    BertTokenizerFast,
+    BertForNextSentencePrediction,
+
+)
+
+from happytransformer.happy_transformer import HappyTransformer
+
+
+class HappyNextSentence(HappyTransformer):
+    """
+    A user facing class for next sentence prediction
+    """
+    def __init__(self, model_type="BERT",
+                 model_name="bert-base-uncased"):
+
+        if model_type == "BERT":
+            model = BertForNextSentencePrediction.from_pretrained(model_name)
+            tokenizer = BertTokenizerFast.from_pretrained(model_name)
+        else:
+            raise ValueError(self.model_type_error)
+        super().__init__(model_type, model_name, model, tokenizer)
+        self._pipeline = None
+        self._trainer = None
+
+    def predict_next_sentence(self, sentence_a, sentence_b):
+        """
+        Determines if sentence B is likely to be a continuation after sentence
+        A.
+        :param sentence_a (string): First sentence
+        :param sentence_b (string): Second sentence to test if it comes after the first
+        :return (float): The probability that sentence_b follows sentence_a
+        """
+        if not self.__is_one_sentence(sentence_a) or not self.__is_one_sentence(sentence_b):
+            raise ValueError('Each inputted text variable for the "predict_next_sentence" method must contain a single sentence')
```

Comment:
I think its weird that we enforce this so strongly. If there is a false positive case, then users have no way to workaround this without modifying their input data (which could affect the transformer output).

Options:
- Remove check (trust users)
- Add option to bypass check (extra param `unsafe=False` or similar)

Reply:
Good call. I also just played around with it and it seems to work well with multiple sentence inputs for both sentence_a and b. 

We'll trust the user.